_docs/migrate_module.md

```diff
@@ -11,13 +11,13 @@ You will have someone by your side in this process. The general flow is to…
 * At that point you can transfer your own repository.
 * If migrating a module from puppetlabs, re-enable github issues.
 * Verify that all webhooks are disabled.
-* Add the module to our [modulesync setup](publish_setup).
+* Add the module to our [modulesync setup][publish_setup].
```

Comment:
This is not working, i don't know why because the other changes are working. Maybe the whitespace?